### PR TITLE
Cy 2470 update kubernetes documentation

### PR DESCRIPTION
**Pull request by AdarShaked**

### CODE REVIEW

File: content/working_with/official_plugins/Orchestration/kubernetes.md [hunk 1]
```diff
@@ -397,3 +485,175 @@ cfy executions start update_resource_definition -d pod -vv -p resource_definitio
...
+this example demonstrates a deployment of eks cluster with one node group.
+We will explain how we used the AWS plugin alongside kubernetes plugin on this example in order to deploy the cluster.
+
+Firstly, in order to create an eks cluster we need to create all the aws node_instances that needed like:
+vpc, subnets , security group, etc. 
+Then, we create the eks cluster and save the kubeconfig content in its runtime properties.
+you can read more about those aws instances in our [AWS plugin documentation]({{< relref "working_with/official_plugins/Infrastructure/aws.md" >}}).
+
+After we have the eks cluster and the node group deployed, we will use the kubernetes plugin to generate the service account authentication token
+(creating service account and rule binding) and create a sanity_master node instance:
```
Comment (EarthmanT) [inline content/working_with/official_plugins/Orchestration/kubernetes.md:506]: `The blueprint defines service account and token node types whose purpose is to generate a token that can be used for simpler plugin authentication.`

File: content/working_with/official_plugins/Orchestration/kubernetes.md [hunk 2]
```diff
@@ -397,3 +485,175 @@ cfy executions start update_resource_definition -d pod -vv -p resource_definitio
...
+          api_key: { get_secret: kubernetes_token }
+          debug: false
+          verify_ssl: false
+    relationships:
+      - type: cloudify.relationships.depends_on
+        target: store_token
+
+{{< /highlight >}}
+
+as you can see, the authentication is done with the api options using the service account token.
```
Comment (EarthmanT) [inline content/working_with/official_plugins/Orchestration/kubernetes.md:525]: No need for `as you can see`.

File: content/working_with/official_plugins/Orchestration/kubernetes.md [hunk 3]
```diff
@@ -346,6 +425,15 @@ node_types:
             payload: V1DeleteOptions
 {{< /highlight >}}
 
+# Using file defined resources 
+As you can see above, kubernetes plugin supports define resources from type : "cloudify.kubernetes.resources.FileDefinedResource".
+This option allows the user to write the kubernetes resource definition in a template yaml file and refer to it from the blueprint.
+like in this [example](https://github.com/cloudify-incubator/cloudify-kubernetes-plugin/blob/master/examples/file-test.yaml).
+
+We are strongly recommend to define your kubernetes resources in that way and not in the "inline" defenition style(our plugin supports both methods).
```
[the remark this answers is not in the document]
Comment (AdarShaked) [reply content/working_with/official_plugins/Orchestration/kubernetes.md:433]: so what actually i need to change? remove last sentence(433)

File: content/working_with/official_plugins/Orchestration/kubernetes.md [hunk 4]
```diff
@@ -195,10 +195,52 @@ One of four methods options can be used to provide the configuration:
...
+    properties:
+      definition:
+        apiVersion: storage.k8s.io/v1beta1
+        kind: StorageClass
+        metadata:
+          name: standard
+        provisioner: kubernetes.io/local
+    relationships:
+      - type: cloudify.kubernetes.relationships.managed_by_master
+        target: kubernetes
```
[the remark this answers is not in the document]
Comment (EarthmanT) [reply content/working_with/official_plugins/Orchestration/kubernetes.md:232]: So also remove this:
```  kubernetes:
    type: cloudify.kubernetes.nodes.Master
    properties:
      configuration: &master_configuration
        file_content: { get_input: configuration_file_content }
```

**Pull request merged**